spec/lib/tasks/eu_exit_business_finder_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'rake'
+require 'rails_helper'
+require 'gds_api/test_helpers/publishing_api_v2'
+require 'gds_api/test_helpers/search'
+
+RSpec.describe 'eu_exit_business_finder:retag_documents_to_facet_values' do
+  include GdsApi::TestHelpers::PublishingApiV2
+  include GdsApi::TestHelpers::Search
+  include PublishingApiHelper
+
+  let(:publishing_api) { Services.publishing_api }
+
+  let(:facet_values_to_replace) do
+    [
+      {
+        "old_facet_value": "old-facet-value",
+        "new_facet_value": "new-facet-value"
+      },
+    ]
+  end
+
+  let(:content_to_update_facet_values) do
+    [
+      {
+        "base_paths": [
+          "/guidance/some-document-about-brexit"
+        ],
+        "old_facet_value": "old-facet-value",
+        "new_facet_value": "new-facet-value"
+      }
+    ]
+  end
+
+  let(:facet_values_to_untag) do
+    [
+      "another-facet-value"
+    ]
+  end
+
+  let(:search_by_facet_value) do
+    {
+      "results": [
+        {
+          "link": "/a-document-about-brexit"
+        }
+      ]
+    }
+  end
+
+  let(:search_by_link) do
+    {
+      "raw_source":
+        {
+          "content_id": "content-id-to-update"
+        }
+    }
+  end
+
+  let(:content_item_links) do
+    {
+      "links":
+      {
+        "facet_values":
+          [
+            "some-facet-value",
+            "old-facet-value",
+            "another-facet-value"
+          ]
+      }
+    }
+  end
+
+  let(:updated_content_item_links) do
+    {
+      "facet_values" =>
```

Comment:
Mixture of styles - this one is using `=>`, the one on line 63 is using `:`. Think the linter prefers the colon.

Reply:
This is deliberate, since `patch_links` requires the `facet_values` key as a string (which `=>` gives you) rather than a symbol (which `:` gives).

Reply:
The keys are a string on line 63 as well. 🤔

Reply:
I've updated the one on line 63 👍

Reply:
That was just one example! I'm surprised the linter hasn't picked up on this.

Reply:
AFAIK the linter like `:` for symbols and `=>` for strings.